### PR TITLE
T22 - Swap Indices

### PR DESCRIPTION
**Pull request by vikrampg1710-tmm**
**T22:**
- Display random series of numbers to the user and ask them to provide two index values which need to be swapped and display the swapped result to the user. 

- For example, 4 8 7 5 6 9 enter the index to be swapped: 2 3 outputs to be: 4 8 5 7 6 9.

### CODE REVIEW

File: Assignments/Program.cs [hunk 1]
```diff
@@ -1,3 +1,81 @@
...
+// ---------------------------------------------------------------------------------------
+// Program.cs
+// T22 - Swap Indices Values
+// ---------------------------------------------------------------------------------------
+using System;
+using System.Collections.Generic;
+using static System.ConsoleColor;
 
-Console.WriteLine("Hello, World!");
+namespace Spark;
```
Comment (JessDarres) [inline Assignments/Program.cs:12]: Why you use this?

File: Assignments/Program.cs [hunk 2]
```diff
@@ -1,3 +1,81 @@
...
+
+public class T22 {
+   public static void Main () {
+      Console.WriteLine ("\x1B[4m" + "Index Swaping Game:-" + "\x1B[0m");
+      Console.WriteLine ("Random series of numbers will be displayed. "
+         + "The user has to enter two indeces value for the values to be swaped.");
+      PerformIndexSwap ();
```
Comment (JessDarres) [inline Assignments/Program.cs]: Get the inputs and checks are in main method after that you call PerformIndexSwap method.

File: Assignments/Program.cs [hunk 3]
```diff
@@ -1,3 +1,81 @@
...
+         }
+         (list[ind1], list[ind2]) = (list[ind2], list[ind1]);
+         Console.WriteLine ("\r\nAfter swaping ==>\r\n");
+         DisplayInBox (list, ind1, ind2);
+         Console.WriteLine ();
+      } else WriteLine ("Program terminated due to incorrect input.", Red);
+
+
+      ///<summary>Displays the input list items using box unicodes</summary>
+      static void DisplayInBox (List<int> list, int ind1, int ind2) {
```
Comment (JessDarres) [inline Assignments/Program.cs]: NIce display.

**Review by JessDarres — CHANGES_REQUESTED** [review]

File: Assignments/Program.cs (hunk 1 above)
Comment (vikrampg1710-tmm) [reply Assignments/Program.cs:12]: Just to indicate this is Spark series task.

**Review by JessDarres — APPROVED** [review]

**Pull request closed without merging**